src/extension.ts

```diff
@@ -48,7 +48,8 @@ export function activate(context: vscode.ExtensionContext) {
         const terminal = findOrCreateTerminal();
 
         terminal.show();
-        terminal.sendText(`test_launcher ${launcher_args}`);
+        const launcherOptions = vscode.workspace.getConfiguration(undefined, null).get('rubyTestLauncher.disableSpring') ? '--disable-spring' : ''
+        terminal.sendText(`test_launcher ${launcher_args} ${launcherOptions}`);
```

Comment:
Come on @darreneng. Where is the unit test coverage for this masterpiece of an extension?

Reply:
lol if you're feeling generous, go ahead and correct `launcher_args` to `launcherArgs`.

The idea of args vs. options here is really not clear (and in it's usage above), but I suppose basically the options are what comes from configuration at this point - I think it's fine as is. 🤷‍♂️

Reply:
tests are for losers :wink:

yeah I agree... my line of thought is one comes from the configuration and the other is a command, but the fact that one of the commands is triggered via `--rerun` makes this pretty confusing.